go/worker/merge/committee/node.go

```diff
@@ -358,7 +358,7 @@ func (n *Node) tryFinalizeResultsLocked(pool *commitment.Pool, didTimeout bool)
 	// NOTE: The roothash backend will start counting its timeout on its own based on
 	//       any received commits so in the worst case the actual timeout will be
 	//       2*roundTimeout.
-	roundTimeout := n.commonNode.Roothash.Info().ComputeRoundTimeout
+	roundTimeout := n.commonNode.GenesisDoc.RootHash.Parameters.RoundTimeout
```

Comment:
This should probably query the current value of the parameters at the current height instead of assuming they will stay the same since genesis.

E.g., maybe add a `ConsensusParameters(context.Context, int64)` to the roothash API (instead of the removed `Info`) and make that query the app? We should probably do the same in other places if we are using the genesis doc directly as the parameters may be mutable eventually.

Reply:
:+1: yeah that was my thinking as well (but then i noticed we just take the value from the doc directly in the txnscheduler case :grin:) will update